src/CGALRenderer.h

```diff
@@ -21,4 +25,14 @@ class CGALRenderer : public Renderer
 	void addGeometry(const shared_ptr<const class Geometry> &geom);
 	const std::list<shared_ptr<class CGAL_OGL_Polyhedron> > &getPolyhedrons() const;
 	void buildPolyhedrons() const;
+	void createPolysets() const;
+	bool last_render_state; // FIXME: this is temporary to make switching between renderers seamless.
+	
+	mutable VertexStates polyset_states;
+	mutable GLuint polyset_vertices_vbo;
+	mutable GLuint polyset_elements_vbo;
```

Comment:
Definitely. I'm assuming this should be called when the renderer is created?

I also wanted to create the renderers once per editor tab, and just swap them as needed instead of the current create/destroy every time the renderer is switched. Not sure, but there are probably states that need to be cleared to allow this. This would be a different change set after this PR is included in the baseline.